examples/ts/src/query_app_data.ts

```diff
@@ -0,0 +1,17 @@
+import {createApi} from "./api";
+
+/**
+ * Example returning data that belongs to application id.
+ */
+async function main() {
+    const api: any = await createApi()
+    const hashBlock = "0x60eaba2542b72a0c17b0e78ad4ae73c3e42e616bb3b5b0dbbc8abfaee5a4aea1";
```

Comment:
Same as earlier. We should probably write data, wait for accept and then query to show the full flow.